manage-mypy.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from subprocess import CalledProcessError, run
+from typing import List, cast
+
+import click
+
+file_path = Path(__file__)
+here = file_path.parent
+exclusion_file = here.joinpath("mypy-exclusions.txt")
+
+
+def write_file(path: Path, content: str) -> None:
+    with path.open(mode="w", encoding="utf-8", newline="\n") as file:
+        file.write(content.strip() + "\n")
+
+
+def get_mypy_failures() -> List[str]:
+    # Get a list of all mypy failures when only running mypy with the template file `mypy.ini.template`
+    command = ["python", "activated.py", "mypy", "--config-file", "mypy.ini.template"]
+    try:
+        run(command, capture_output=True, check=True, encoding="utf-8")
+    except CalledProcessError as e:
+        if e.returncode == 1:
+            return cast(List[str], e.stdout.splitlines())
+        raise click.ClickException(f"Unexpected mypy failure:\n{e.stderr}") from e
+    return []
+
+
+def build_exclusion_list(mypy_failures: List[str]) -> List[str]:
+    # Create content for `mypy-exclusions.txt` from a list of mypy failures
+    return sorted(
+        {".".join(Path(line[: line.find(".py")]).parts) for line in mypy_failures[0 : len(mypy_failures) - 1]}
+    )
+
+
+@click.group()
+def main() -> None:
+    pass
+
+
+@main.command()
+@click.option("--check-exclusions/--no-check-exclusions", show_default=True, envvar="CHIA_MANAGE_MYPY_CHECK_EXCLUSIONS")
+def build_mypy_ini(check_exclusions: bool = False) -> None:
```

Comment:
Might make sense for the command to just be `build`?  `build-mypy-ini` feels verbose for a single subcommand situation.

Reply:
```
Commands:
  build-exclusions
  build-mypy-ini
```